plugins/modules/rabbitmq_vhost.py

```diff
@@ -39,53 +39,171 @@
     type: str
     default: present
     choices: [present, absent]
-'''
+  login_user:
+      description:
+          - RabbitMQ user for connection.
+      type: str
+      default: guest
+  login_password:
+      description:
+          - RabbitMQ password for connection.
+      type: str
+      default: guest
+  login_host:
+      description:
+          - RabbitMQ host for connection.
+      type: str
+  login_port:
+      description:
+          - RabbitMQ management API port.
+      type: str
+      default: '15672'
+  login_protocol:
+      description:
+          - RabbitMQ management API protocol.
+      type: str
+      choices: [ http , https ]
+      default: http
+  ca_cert:
+      description:
+          - CA certificate to verify SSL connection to management API.
+      type: path
+      aliases: [ cacert ]
+  client_cert:
+      description:
+          - Client certificate to send on SSL connections to management API.
+      type: path
+      aliases: [ cert ]
```

Comment:
```suggestion
```
Please remove them from the code too.
Justification: we shouldn't use aliases in general. only in cases of really inconsistent initial naming or typos.